src/main/java/com/fryrank/validator/ReviewValidator.java

```diff
@@ -0,0 +1,37 @@
+package com.fryrank.validator;
+
+import org.springframework.validation.Errors;
+import com.fryrank.model.Review;
+import org.springframework.validation.Validator;
+
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+
+public class ReviewValidator implements Validator {
+    public static final String ISO_DATE_TIME = "isoDateTime";
+    public static final String ISO_DATE_TIME_REJECTION_REQUIRED_CODE = "field.required";
+    public static final String ISO_DATE_TIME_REJECTION_FORMAT_CODE = "field.invalidFormat";
+    public static final String ISO_DATE_TIME_REJECTION_FORMAT_REASON = "The provided isoDateTime is not in ISO format.";
+    @Override
+    public boolean supports(Class clazz) {
+        return Review.class.isAssignableFrom(clazz);
+    }
+
+    @Override
+    public void validate(Object target, Errors errors) {
```

Comment:
Can you add @NonNull to the method params here?

Reply:
Done in [c52da6f](https://github.com/NickPriv/FryRankBackend/pull/62/commits/c52da6f116a93053a61ae696f15c2504a2a2a5c4).